openedx/features/edly/api/tests/test_serializers.py

```diff
@@ -75,3 +86,22 @@ def test_site_data(self):
             assert color_value == serializer.data['site_data'].get(color_key)
 
         assert self.edly_sub_org_of_user.lms_site.name == serializer.data['site_data'].get('display_name')
+
+        protocol = 'https' if self.request.is_secure() else 'http'
+        url = self.test_site_configuration['SITE_NAME']
+        expected_site_url = '{}://{}'.format(protocol, url) if url else ''
+        assert expected_site_url == serializer.data['site_data'].get('site_url')
```

Comment:
We should add the generated url validation as well here like
https://www.kite.com/python/answers/how-to-check-if-a-string-is-a-url-in-python

Reply:
Good suggestion, adding it here.